src/99-appendices/01-contributors.md

```diff
@@ -127,13 +142,17 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 -   Guiomar Niso 📖💡📢
```

Comment:
```suggestion
-   Guiomar Niso 📖💡📢 🤔 🎨 🔍 👀 📋 📝 🔧🐛 💻 🔣 ✅ 💬 
```
Long time without updating this, thanks!

Reply:
@guiomar Can you just add your updated icons on https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors? Those will get added in when we release.

Reply:
Sure! Thanks a lot

Reply:
@effigies If @guiomar will update on the link you gave, what can I do with this 'suggested change'?

Reply:
No need to do anything, here. You can just resolve the conversation.

Reply:
@guiomar I added your "new" contribution emojis to the Wiki, it seems like you forgot.

Reply:
@effigies I don't see 'resolve'. Maybe I don't have the rights? Or am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/9084751/98733463-d62dd700-23a0-11eb-9929-bab9d3e4e1c2.png)


you may not have had sufficient rights until now, but I just gave you more rights. Can you see the button now?

Reply:
Thanks @sappelhoff!! I think I just didn't do it right, thanks for checking